test/Confluent.Kafka.IntegrationTests/Tests/AddBroker.cs

```diff
@@ -14,6 +14,8 @@
 //
 // Refer to LICENSE for more information.
 
+#pragma warning disable xUnit1026
```

Comment:
disable warn on unused parameters (the way config parameters are injected in results in this when they are unused by a particular test).